module/ptu.js

```diff
@@ -381,6 +381,16 @@ function registerHandlebars() {
 
   Handlebars.registerHelper("tmName", function (tmNum) { return game.ptu.TMsData.get(tmNum) });
 
+  Handlebars.registerHelper("totalWealth", function (actor) {
+    let total = actor.data.data.money;
+    actor.inventory.forEach(item => {
+      if (item.data.type = 'item') 
+        if(!isNaN(item.data.cost))
+          total += (item.data.cost * item.data.quantity);
+    })
+    return total
+  });
+
```

Comment:
Not a fan of handling this as a handlebar helper, now this is done during rendering, instead make this call in the character-sheet.js's getData function. 

Also please note that `actor.inventory` is something locally created for characters and not a foundry object, and therefore should be avoided being used outside of sheet rendering.

I'd personally also use a cleaner loop, but that's pure preference.
```js
actor.itemTypes.item.reduce((total, item) => total + (!isNaN(item.data.data.cost) ? item.data.data.cost * (item.data.data.quantity ?? 0) : 0), actor.data.data.money);
```